lib/ClimaCoreMakie/LICENSE.md

```diff
@@ -0,0 +1,13 @@
+Copyright 2021 Climate Modeling Alliance
```

Comment:
This should be a `NOTICE` file, and the full apache license should be in `LICENSE` (see ClimaCore or ClimaCoreVTK)

Reply:
should be fixed now